docs/topics/data-analysis/data-analysis-work-with-data-sources.md

```diff
@@ -1,20 +1,124 @@
 [//]: # (title: Retrieve data from files)
 
-Simple guide how to start working data sources with Kotlin notebooks.
+Kotlin Notebook, coupled with the  [DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html), enables 
+you to work with both non-structured and structured data. It offers the flexibility to transform non-structured data, 
+such as those found in TXT files, into structured datasets. For this transformation, you can utilize methods 
+such as [`add`](https://kotlin.github.io/dataframe/adddf.html), [`split`](https://kotlin.github.io/dataframe/split.html),
+[`convert`](https://kotlin.github.io/dataframe/convert.html), and [`parse`](https://kotlin.github.io/dataframe/parse.html). 
+Additionally, this toolset enables the retrieval and manipulation of data from various structured file formats, 
+including CSV, JSON, XLS, and XLSX.
```

Comment:
We also read from the format Apache Arrow

Reply:
thank you! - I will include it here! :)